Android/Fundamentals/drawables-ii/create-layer-list.md

```diff
@@ -0,0 +1,123 @@
+# create-layer-list
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: how to
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+LayerList Drawable – manages an array of drawables. These are drawn in an array order, the largest element with the largest index is drawn on top. For a simple example, the following layer list draws several shapes in relation to each other:
+
+Step 1: Create a layer list xml file and place it in the res/drawable folder
```

Comment:
detail these steps a bit more, remove boilerplate, try to reduce the code to just the most pertinent bits, link to docs for full examples